### PR TITLE
include JS files for webpack assets in foreman-assets package

### PR DESCRIPTION
**Pull request by mmoll**
this should be built into:

* [x] Nightly
* [x] 1.18

I'm unsure what elso needs to get packaged now for the plugin asset builds with webpack to succeed.

### CODE REVIEW

**Comment by ekohl** [comment]
We modified the rake task that precompiles assets so as long as that's called, it should work. It should be noted that some plugins have their own nodejs deps and I don't know how you'd handle that.

**Comment by mmoll** [comment]
this allows now to build _rex 1.5.3 https://ci.theforeman.org/job/packaging_build_deb_plugin/1832/

**Comment by mmoll** [comment]
@ekohl @ehelms @dLobatog @sharvit any more comments? :)

File: debian/stretch/foreman/foreman-assets.postinst [hunk 1]
```diff
@@ -29,6 +29,9 @@ fi
 # Own all the core files
 chown -Rf foreman:foreman '/usr/share/foreman'
 
+# make webpack file executable
+chmod 755 '/usr/share/foreman/node_modules/webpack/bin/webpack.js'
```
Comment (sharvit) [inline debian/stretch/foreman/foreman-assets.postinst]: Shouldn't it live under `./node_modules/.bin/webpack`?

Comment (mmoll) [reply debian/stretch/foreman/foreman-assets.postinst]: That's a symlink to the actiual file - at least in the environment, this got built on.

**Comment by mmoll** [comment]
> It should be noted that some plugins have their own nodejs deps and I don't know how you'd handle that.

I don't know

**Pull request merged**